.travis.yml

```diff
@@ -45,19 +45,33 @@ jobs:
         - composer require --dev infection/infection
         - composer test:mutations
     - stage: test
-      name: 'ZF2 Integration'
+      name: 'ZF2 Integration, DEPS=LATEST'
       php: 7.1
       script:
         - cd test/ZF2
         - composer install --no-interaction --prefer-source
         - vendor/bin/phpunit
     - stage: test
-      name: 'ZF3 Integration'
+      name: 'ZF2 Integration, DEPS=LOWEST'
+      php: 7.1
+      script:
+        - cd test/ZF2
+        - composer update --prefer-lowest --no-interaction --prefer-source
```

Comment:
Why source?

Reply:
copy/paste from other stages. shall i remove it?

Reply:
from https://getcomposer.org/doc/03-cli.md

> --prefer-source: There are two ways of downloading a package: source and dist. For stable versions Composer will use the dist by default. The source is a version control repository. If --prefer-source is enabled, Composer will install from source if there is one. This is useful if you want to make a bugfix to a project and get a local git clone of the dependency directly.

seems like it should be removed from all stages to me...

Reply:
cool, `--prefer-source` isn't necessary in our build:
https://github.com/enlitepro/enlite-monolog/pull/64

i'll remove it from this PR as well.

Reply:
removed.